Taskfile.yml

```diff
@@ -15,7 +15,9 @@ vars:
   G_PACKAGE_VENV_DIR: "{{.G_BUILD_DIR}}/package-venv"
   G_WEBUI_BUILD_DIR: "{{.G_BUILD_DIR}}/webui"
   G_WEBUI_NODEJS_BUILD_DIR: "{{.G_BUILD_DIR}}/webui-nodejs"
+  G_LOG_VIEWER_WEBUI_BUILD_DIR: "{{.G_BUILD_DIR}}/log-viewer-webui"
   G_WEBUI_NODEJS_BIN_DIR: "{{.G_WEBUI_NODEJS_BUILD_DIR}}/bin"
+  G_LOG_VIEWER_WEBUI_BIN_DIR: "{{.G_NODEJS_22_DIR}}/bin"
```

Comment:
For consistency, let's name this as `G_NODEJS_22_BIN_DIR`.

Reply:
I'm not opposed to `node-webui`, but having `node` and `node-webui` might be a bit confusing for someone unfamiliar with the issue. How about `node-22` and `node-14`? The start-clp code will tell the dev which is which.

Reply:
how about `node` and `node-14`? unless we require the version to be 22 rather than any latest version

Reply:
Yeah, `node-14` is specifically for the webui since it doesn't support newer node versions. That said, I think it's still going to be confusing for someone new to see `node` and `node-14` or `node-default` and `node-14`. I think it's a small temporary cost to call it `node-22` for now and then we can switch to a single `node` once we switch webui frameworks.

Reply:
Sure